json-schemas/src/account-stats.json

```diff
@@ -0,0 +1,802 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Ably Account Statistics Message",
+  "type": "object",
+  "properties": {
+    "accountId": {
+      "type": "string",
+      "description": "The ID of the Ably account the statistics are for."
+    },
+    "intervalId": {
+      "type": "string",
+      "description": "time period for statistics in date-time format yyyy-mm-dd:hh:mm:ss"
+    },
+    "unit": {
+      "type": "string",
+      "description": "Unit of time for these statistics from the intervalId forwards.",
+      "pattern": "^minute$"
+    },
+    "schema": {
+      "type": "string",
+      "description": "URI to this schema."
+    },
+    "entries": {
```

Comment:
can I assume 1 single entry per account stats entry, i.e. no need to flatten an array, similarly to the App stats?

Reply:
correct